uvm/api/com/untangle/uvm/app/IPMatcher.java

```diff
@@ -312,6 +312,13 @@ private void initialize(String matcher)
 
                 this.rangeMin = addrToLong(addrMin);
                 this.rangeMax = addrToLong(addrMax);
+
+                // Swap min and max if they are in reverse order
+                if (this.rangeMin > this.rangeMax) {
```

Comment:
@cblaise-untangle should we keep this change. It can be useful in `isMatch()` method which looks for the input IP in min and max range,
```
        case RANGE:
            tmp = addrToLong(address);
            return ((this.rangeMin <= tmp) && (tmp <= this.rangeMax));

Reply:
Let's leave it out.  Given more time to think about it, I don't think it makes sense to spend extra time in such a high use area doing that.